### PR TITLE
Rename radial search parameters score and distance to min_score and max_distance

### PR DESCRIPTION
**Pull request by junqiu-lei**
### Description
Rename radial search parameters score and distance to min_score and max_distance
 
### Issues Resolved
Part of  https://github.com/opensearch-project/k-NN/issues/814

### Check List

- [x] Commits are signed as per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/k-NN/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: src/main/java/org/opensearch/knn/index/query/KNNQueryBuilder.java [hunk 1]
```diff
@@ -348,12 +348,12 @@ public int getK() {
         return this.k;
     }
 
-    public float getDistance() {
-        return this.distance;
+    public float getMax_distance() {
```
Comment (navneet1v) [inline src/main/java/org/opensearch/knn/index/query/KNNQueryBuilder.java:351]: rename this to getMaxDistance

File: src/main/java/org/opensearch/knn/index/query/KNNQueryBuilder.java [hunk 2]
```diff
@@ -348,12 +348,12 @@ public int getK() {
...
+        return this.max_distance;
     }
 
-    public float getScore() {
-        return this.score;
+    public float getMin_score() {
```
Comment (navneet1v) [inline src/main/java/org/opensearch/knn/index/query/KNNQueryBuilder.java:355]: rename the function to getMinScore

File: src/main/java/org/opensearch/knn/index/query/KNNQueryBuilder.java (hunk 1 above)
Comment (junqiu-lei) [reply src/main/java/org/opensearch/knn/index/query/KNNQueryBuilder.java]: Updated.

File: src/main/java/org/opensearch/knn/index/query/KNNQueryBuilder.java (hunk 2 above)
Comment (junqiu-lei) [reply src/main/java/org/opensearch/knn/index/query/KNNQueryBuilder.java]: Updated.

**Review by navneet1v — APPROVED** [review]

**Review by naveentatikonda — APPROVED** [review]

**Pull request merged**